### ISSUE TITLE
WebGL crash, to much textures or bad hardware?

### ISSUE THREAD

**Issue by Inateno** [pixijs/pixi.js#4759]
Hi,
I'm running an issue I can't reproduce on my computer but some players reported it.

Here is a screen https://imgur.com/a/dDBWw
To resolve this they have to restart the browser (sometimes the computer, not really sure about it).

It happens after the game has been loaded (I use the default PIXI.Loader).

And ofc I never got this issue on my computer (pretty strong hardware, maybe this is the difference?).

How can I know if this an issue from Chrome (or Firefox), WebGL, hardware?
Has anyone here already encounter this issue and solved it? How?

Thanks.

**Comment by ivanpopelyshev** [comment]
Unfotunately, my telepathy is worthless here. Need more informatin.

**Comment by emilekberg** [comment]
To check the browser/vendor graphics support, you can have them check http://webglreport.com/?v=1. 
What may also help is to have them check the browser dev console, to see if any errors are thrown. 
Also it's quite difficult to figure out what's the error since there is no picture of what the game SHOULD look like.

**Comment by Inateno** [comment]
@ivanpopelyshev my bad --' : 4.5.6

@emilekberg most of the players don't understand what you are talking about ;) but I have one who opened the console and nothing.

Rebooting the browser and the game was running well.
Here is what the game looks like: https://www.youtube.com/watch?v=VbifHwyqfR0

**Comment by ivanpopelyshev** [comment]
We kinda fixed critical issue in 4.6.1

**Comment by Inateno** [comment]
Ok I'll try an update.

**Comment by themoonrat** [comment]
@Inateno closing as I'm assuming newer PIXI versions solved your issue. If not, please re-open :)

**Comment by Inateno** [comment]
Yep, so far no "bug feedback" anymore (I got only 3-4 like those so not happening all the time).

Mb it's still happening but no-one told me, will see.

Ah also, I got the exact same bug on the Nintendo Switch browser (launching the browser with the "dns hack") but this was happening all the time, maybe the Switch doesn't have enough memory and so the bug is happening all the time but it is not debuggable!

Will re-open if I have more information about this.
